Help/FAQ.rst

```diff
@@ -36,13 +36,13 @@ How do I get started with coala?
 --------------------------------
```

Comment:
@Dayfiri are you sure the commit body is correct? :) Please take a look at our commit guidelines too: http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html

You can change the commit message with `git commit --amend`.